websites/A/AniTube.in/presence.ts

```diff
@@ -0,0 +1,98 @@
+    { pathname } = document.location;
+
+  if (searchContainer.style.display === "block") {
+    presenceData.details = `Searching for ${searchQuery}`;
+    presenceData.state = "";
+  } else {
+    if (pathname === "/") {
+      presenceData.details = "Browsing Home Page";
+      presenceData.state = "";
+    } else if (pathname === "/busca.php") {
+      const searchBox: HTMLDivElement = document.querySelector(
+          ".mContainer_title_small_content"
+        ),
+        searchQuery: string = searchBox.innerText;
+      presenceData.details = "Looking at search results for";
+      presenceData.state = searchQuery.substring(17);
+    } else if (
+      pathname === "/anime" ||
+      pathname === "/animes-dublado" ||
+      pathname === "/doramas" ||
+      pathname === "/tokusatsu" ||
+      pathname === "/donghua"
+    ) {
+      presenceData.details = "Looking at all Animes";
+      presenceData.state = "";
+    } else if (
+      pathname.startsWith("/anime/letra/") ||
+      pathname.startsWith("/animes-dublado/letra/") ||
+      pathname.startsWith("/doramas/letra/") ||
+      pathname.startsWith("/tokusatsu/letra/") ||
+      pathname.startsWith("/donghua/letra/")
+    ) {
+      const query: string = pathname.substring(pathname.length - pathname.split("").reverse().join("").indexOf("/"));
+      if (query !== "todos") {
+        presenceData.details = "Looking for Animes";
+        presenceData.state = `starting with ${query}`;
+      } else {
+        presenceData.details = "Looking at all Animes";
+        presenceData.state = "";
+      }
+    } else if (
+      pathname.startsWith("/anime/") ||
+      pathname.startsWith("/animes-dublado/") ||
+      pathname.startsWith("/doramas/") ||
+      pathname.startsWith("/tokusatsu/") ||
+      pathname.startsWith("/donghua/")
+    ) {
+      const titleElement: HTMLDivElement = document.querySelector(
+          ".anime_container_titulo"
+        ),
+        title: string = titleElement.innerText;
+      presenceData.details = "Checking synopsis";
+      presenceData.state = title;
+    } else if (pathname.startsWith("/video/")) {
+      const titleElement: HTMLDivElement = document.querySelector(
+          ".mContainer_title_small_content"
+        ),
+        title: string = titleElement.innerText,
+        videoElement: HTMLVideoElement = document.querySelector("video"),
+        paused: boolean = videoElement === null ? null : videoElement.paused,
+        duration: number = videoElement !== null ? videoElement.duration : NaN;
+
+      presenceData.details = `Watching ${title.substr(9, title.substr(9).indexOf("ep"))}`;
+      presenceData.state = `Episode ${parseInt(title.substr(title.indexOf("ep") + 2))}`;
+      presenceData.buttons = [{
+        label: "Watch Along",
+        url: document.location.href
+      }];
+      if (!(paused === null || paused || isNaN(duration))) {
+        presenceData.state = "Time left";
```

Comment:
for what purpose????????

Reply:
Sorry, this was before I was aware of `getTimestampsfromMedia`, removed.